order-service/src/main/resources/docker-database.yml

```
@@ -22,7 +22,7 @@ spring:
     database-platform: org.hibernate.dialect.MySQL8Dialect
     hibernate:
       ddl-auto: update
-    properties:
-      hibernate:
-        format_sql: true
-        show_sql: true
\ No newline at end of file
+#    properties:
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)